ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-radius-source-interfaces.j2

```diff
@@ -0,0 +1,18 @@
+{% if ip_radius_source_interfaces is defined and ip_radius_source_interfaces is not none %}
+
+## IP RADIUS Source Interfaces
+
+### IP RADIUS Source Interfaces
+
+| VRF | Source Interface Name |
+| --- | --------------- |
+{%     for ip_radius_source_interface in ip_radius_source_interfaces %}
+| {% if ip_radius_source_interface['vrf'] is defined and ip_radius_source_interface['vrf'] is not none %} {{ ip_radius_source_interface['vrf'] }} {% else %}  default {% endif %} | {{ ip_radius_source_interface['name'] }} |
```

Comment:
Please use `ip_radius_source_interface['vrf'] | arista.avd.default('default')` instead of `{% if ip_radius_source_interface['vrf'] is defined and ip_radius_source_interface['vrf'] is not none %} {{ ip_radius_source_interface['vrf'] }} {% else %}  default {% endif %}`

Reply:
Changes have been implemented as stated above...